### ISSUE TITLE
Add steps to uninstall brew cask in USAGE.md

### ISSUE THREAD

**Issue by nunocalaim** [Homebrew/homebrew-cask#59154]
### Description of feature/enhancement

USAGE.md should have instructions on how to completely uninstall brew cask and all the casks involved. 

### Justification

I cannot run list, outdated, install with brew cask as there is an undefined method 'release' for OS::Mac:Module. And I would like to completely wipe brew cask and reinstall it again

### Example use case

When someone decides that he doesn't want to use brew cask anymore or wants to do a fresh installation

**Comment by nunocalaim** [comment]
by the way: How can I completely uninstall brew cask? and all the applications?

**Comment by nunocalaim** [comment]
Thank you for the quick explanation. That solved the issue temporarily (!?) I have attempted to uninstall all the applications using your command but I get:

Error: Cask 'evernote' is unreadable: undefined method `release' for OS::Mac:Module

Then I run the fix in #58046. but this time it doesn't solve the issue.

Can I simply remove the folder located in  opt/homebrew-cask/Caskroom ?

**Comment by vitorgalvao** [comment]
> Can I simply remove the folder located in  opt/homebrew-cask/Caskroom ?

You can, but that won’t uninstall anything. Things will still be left in your system. Do this:

```bash
for cask in $(brew cask list); do
  brew cask uninstall --force "${cask}"
done
```

It will uninstall the ones it can. For the others, you’ll have to go the manual route.
